serial.md

```diff
@@ -4,7 +4,13 @@ summary: The SERIAL data type defaults to a unique 64-bit signed integer that is
 toc: false
 ---
 
-The `SERIAL` [data type](data-types.html) defaults to a unique 64-bit signed integer. The default value is the combination of the insert timestamp and the ID of the node executing the insert. This combination is guaranteed to be globally unique. Also, because value generation does not require talking to other nodes, it is much faster than sequentially auto-incrementing a value, which requires distributed coordination.
+The `SERIAL` [data type](data-types.html) is a column data type which
```

Comment:
Thanks for strengthening this! While you're here, can you rename the `Format` section to `Syntax`?